src/converters/number-compare-converters.js

```diff
@@ -1,3 +1,27 @@
+/**
+ * ## Number Comparison Converters
+ *
+ * Converters allow you to convert data -- in particular, model variables that you display in your project's user interface -- from one form to another.
+ *
+ * For a number comparison converter, the original format is your model variable, and the resulting "format" is a (possibly unrelated) value of your choosing. This resulting value is selected based on how the value of the model variable compares to a reference value that you pass to the converter.
+ *
+ * There are two ways to specify conversion or formatting for the display output of a particular model variable:
+ *
+ * * Use the `|` (pipe) character within the value of any `data-f-` attribute.
+ * * Add the attribute `data-f-convert` to any element that also has the `data-f-bind` or `data-f-foreach`.
+ *
+ * For example:
+ *
+ *      <!-- displays "true" or the number of widgets -->
+ *      <span data-f-value="widgets | greaterThan(50)"></span>
+ *      <span data-f-value="widgets" data-f-convert="greaterThan(50)"></span>
```

Comment:
don't think this is useful as a `data-f-convert` (unless you want to compare every bound element within this as to 50 which seems a stretch as a use-case), so we can remove that.

Also should be bind, not value